### ISSUE TITLE
Tuff Golem Crash

### ISSUE THREAD

**Issue by ScratchyFireGithub** [Faboslav/friends-and-foes#188]
### Minecraft version information

1.20.1

### Mod loader information

Forge

### Mod loader version information

Forge 47.2.0. (Arclight Server)

### Mod version information

2.0.7

### Reproduction Steps

My server randomly crashes. I looked through the logs, and I know the issue.

TL;DR - The issue is this:
com.faboslav.friendsandfoes.entity.TuffGolemEntity.setPoseWithoutPrevPose
This code is run in a Non-GL thread.

### Crash Report and latest.log

TL;DR - The issue is this:
com.faboslav.friendsandfoes.entity.TuffGolemEntity.setPoseWithoutPrevPose
This code is run in a Non-GL thread.

Full Crash Report:

[crash.log](https://github.com/Faboslav/friends-and-foes/files/13923106/crash.log)

### Additional information

The issue is this:
com.faboslav.friendsandfoes.entity.TuffGolemEntity.setPoseWithoutPrevPose
This code is run in a Non-GL thread.

**Comment by Faboslav** [comment]
If you are running on some kind of bukkit server, there is probably your cause.

**Comment by ScratchyFireGithub** [comment]
Yeah, I’m running Arclight. Still, idk if this can be fixed, but I hope it can. Kind of inconvenient. If not, it’s alright.

**Comment by Faboslav** [comment]
I looked into this little bit more, nothing i can do about this on my side, even the `EntityPoseChangeEvent` is purely bukkit (spiggot) only thing, so you should probably try to report this to their github, so they can somehow fix it.
